src/pages/Tasks.jsx

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+
+import './styles/Table.css';
+
+import { Hamburger } from '../components/Buttons/Hamburger/Hamburger';
+import { Button } from '../components/Buttons/Button/Button';
+
+import startIcon from '../assets/images/startIcon.png';
+import deadlineIcon from '../assets/images/deadlineIcon.png';
+
+const pageName = 'Tasks';
+const menuItems = ['Name', 'Start', 'Deadline', 'Actions'];
+const TasksBody = [
+  {
+    name: 'Create the DB',
+    start: '28.01.2021',
+    start_img: startIcon,
+    deadline: '28.01.2021',
+    deadline_img: deadlineIcon,
+    buttons: 'buttons',
+  },
+  {
+    name: 'Implement the props',
+    start: '28.01.2021',
+    start_img: startIcon,
+    deadline: '28.01.2021',
+    deadline_img: deadlineIcon,
+    buttons: 'buttons',
+  },
+];
+
+export function Tasks(props) {
+  const { showDrawer, toggle } = props;
+  return (
+    <article>
+      <header className='header'>
+        <Hamburger showDrawer={showDrawer} toggle={toggle} />
+        <Button name={pageName} />
+      </header>
+      <p className='page-name'>
+        {pageName}
+        <span>{`(${TasksBody.length})`}</span>
```

Comment:
```suggestion
        <span>{TasksBody.length}</span>
```
it looks better

Reply:
Like in previous case i need brackets here

Reply:
```suggestion
     <span>({ProgressBody.length})</span>
```
it looks more simple

Reply:
Changed like in previous case